internal/cli/status.go

```diff
@@ -101,6 +102,15 @@ func (c *StatusCommand) Run(args []string) int {
 		c.ui.Output(wpAppFlagAndTargetIncludedMsg, terminal.WithWarningStyle())
 	}
 
+	// Optionally refresh status
+	if c.flagRefreshAppStatus {
+		if err := c.RefreshApplicationStatus(projectTarget, appTarget); err != nil {
```

Comment:
Potential optimization: if they specify a -app flag, we don't need to refresh status on every app in the project.

Reply:
That should happen already!